tests/build_rpms.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+# Copied from https://github.com/htcondor/htcondor-ce
+
+set -exu
+
+# Build source and binary RPMs.
+# SRPMs will be /tmp/rpmbuild/SRPMS/*.rpm.
+# Binary RPMs will be /tmp/rpmbuild/RPMS/*/*.rpm.
+
+OS_VERSION=$1
+BUILD_ENV=$2
+
+# Create the condor user/group for subsequent chowns,
+# using the official RHEL UID/GID
+groupadd -g 64 -r condor
+useradd -r -g condor -d /var/lib/condor -s /sbin/nologin \
+        -u 64 -c "Owner of HTCondor Daemons" condor
+
+if  [[ $OS_VERSION == 7 ]]; then
+    YUM_PKG_NAME="yum-plugin-priorities"
+else
+    YUM_PKG_NAME="yum-utils"
+fi
```

Comment:
¯\_(ツ)_/¯ maybe! That's something to put in our back pocket since that's what we're doing in [software-base](https://github.com/opensciencegrid/docker-software-base/blob/master/Dockerfile#L14-L22), too